docs/markdowns/quickstart.md

```diff
@@ -54,57 +104,5 @@ info['features'] = features
 info['target'] = target
 ```
 
-What do we care about? We care about the model, the data, the metrics and the code.
-
-### Setup
-
-Let's [create a new repo](https://xethub.com/xet/create) in the UI.
-
-We clone the repo to our local filesystem, saving everything we want, and committing it:
-
-```bash
-git xet clone https://xethub.com/user/repo
-```
-
-```python
-import pyxet
-import json
-import joblib
-
-
-df.to_csv(f"titanic.csv",
-          index=False)
-
-with open("info.json") as f:
-    json.dump(info, f)
-
-with open("model.pkl") as f:
-    joblib.dump(model, f)
-```
-```bash
-git add .
-git commit -m "first commit"
-git push
-```
-
-Of course you can save your code as well, and upload it with the command:
-> This will work no matter the size of your data, model or logs.
-
-## Next steps
-
-Do you want to experiment with another model?   
-you can clone the repo, create a new branch and try a different model.
-
-* All models will be saved, managed and versioned using git.
-* All metrics and logs will be saved such that you can compare them easily.
-* You can share your repo with your team and collaborate on it.
-    * Sharing data
-    * Pushing code
-    * Running experiments
-    * Saving models
-    * Saving logs
-
-Checkout this [titanic app](https://xethub.com/xdssio/titanic-server-example), for a more comprehensive example.
-
-
+Continue to see how you can use pyxet on a private XetHub repository.
```

Comment:
Continue to where?

Reply:
<img width="1127" alt="image" src="https://user-images.githubusercontent.com/1296134/235214540-6242e2e7-a9a1-48da-b5ca-ed7ecf0166bc.png">

I'll add "to the next section" to help clarify.